api/redisfailover/v1/types.go

```diff
@@ -72,3 +76,20 @@ type RedisFailoverList struct {
 
 	Items []RedisFailover `json:"items"`
 }
+
+// ImagePullPolicy defines the pull policy with a default of Always
+type ImagePullPolicy corev1.PullPolicy
+
+// UnmarshalJSON sets the default value to Always on JSON decode
+func (e *ImagePullPolicy) UnmarshalJSON(b []byte) error {
```

Comment:
```suggestion
func (i *ImagePullPolicy) UnmarshalJSON(b []byte) error {
```
Convention in go is use the first leter or leters of the type :)